enumerable.rb

```diff
@@ -0,0 +1,88 @@
+module Enumerable
+  def my_each
+    count = 0
+    while count < self.size
+      yield(self[count])
+      count += 1
+    end
+
+  def my_each_width_index
+    index = 0
+    for element in self
+      yield(self[element], element)
+      index += 1
+  end
+
+  def my_select
+    result= []
+    self.my_each{|element|
+      if yield(element)
+        result << element
+      end
+    }  
+  end
+
+  def my_all?
+    result = false
+    self.my_each{|element| yield(element) ? result = true : result= false}
+    result
+  end
+
+  def my_any?
+    result = false
+    self.my_each{|element| result = true if yield(element)}
+  end
+

```

Comment:
`my_none?` also doesn't return anything. Also we need to return `false` inside the block once `yield(element)`

Reply:
done

Reply:
`my_all`  :+1: